Automattic/WP-Job-Manager
# Update language; hide expired job listings from archives

Fixes #1884

### Changes proposed in this Pull Request

* Updates language between the 'Filled' and 'Expired' options to be more consistent.
* Excludes expired and filled jobs from WP Search and Jobs' archives, widgets, and feed.

Search exclusions are built off of the changes in reverted [PR 1570](https://github.com/Automattic/WP-Job-Manager/pull/1570). In this instance, additional `meta_query` checks were added to ensure that specified meta were not only the checked value, but that the meta existed at all.

This latter check, missing originally, seems to be why 'normal' posts and pages were excluded from search. With the changes made here, that content returns in search as expected, even if filled and hidden posts are hidden.

### Testing instructions

* Check out the branch.
* Navigate to **Job Listings --> Settings --> Job Listings** and enable or disable the `Filled Positions` and `Expired Positions` options, checking WP Search, Sitemap, and widgets like the 'Recent Jobs' widget.
